docs/fr/docs/index.md

```diff
@@ -27,82 +23,82 @@
+* **Rapide**: Très haute performance, à égalité avec **NodeJS** et **Go** (merci à Starlette et Pydantic). [L'un des framework Python les plus rapides disponibles](#performance).
 
-* **Fast to code**: Increase the speed to develop features by about 200% to 300%. *
-* **Fewer bugs**: Reduce about 40% of human (developer) induced errors. *
-* **Intuitive**: Great editor support. <abbr title="also known as auto-complete, autocompletion, IntelliSense">Completion</abbr> everywhere. Less time debugging.
-* **Easy**: Designed to be easy to use and learn. Less time reading docs.
-* **Short**: Minimize code duplication. Multiple features from each parameter declaration. Fewer bugs.
-* **Robust**: Get production-ready code. With automatic interactive documentation.
-* **Standards-based**: Based on (and fully compatible with) the open standards for APIs: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (previously known as Swagger) and <a href="http://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
+* **Rapide à coder**: Augmenter la vitesse de développement des fonctionnalités d'environ 200 % à 300 %. *
+* **Moins de bugs**: Réduire d'environ 40 % les erreurs humaines (de développement). *
+* **Intuitif**: Excellente compatibilité avec les IDE. <abbr title="également connu sous le nom d'auto-complétion, autocomplétion, IntelliSense">Completion</abbr> complète. Moins de temps pour le débogage.
+* **Facile**: Conçu pour être facile à utiliser et à apprendre. Moins de temps à lire la documentation.
+* **Concit**: Minimise la duplication des codes. Plusieurs fonctionnalités pour chaque déclaration de paramètres. Moins de bogues.
+* **Robuste**: Obtenez un code prêt pour la production. Avec une documentation interactive automatique.
+* **Basé sur des normes**: Basé sur (et entièrement compatible avec) les normes ouvertes pour les API: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (précédemment connu sous le nom de Swagger) et <a href="http://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
 
-<small>* estimation based on tests on an internal development team, building production applications.</small>
+<small>* estimation basée sur des tests sur une équipe de développement interne, construisant des applications de production.</small>
 
 ## Opinions
 
-"_[...] I'm using **FastAPI** a ton these days. [...] I'm actually planning to use it for all of my team's **ML services at Microsoft**. Some of them are getting integrated into the core **Windows** product and some **Office** products._"
+"_[...] J'utilise énormément **FastAPI** ces jours-ci. [...] En fait, je compte l'utiliser pour tous mon equipe chez **Microsoft**. Certains d'entre eux sont intégrés dans le produit de base **Windows** et dans certains produits **Office**._"
```

Comment:
```suggestion
"_[...] J'utilise énormément **FastAPI** ces jours-ci. [...] En fait, je compte l'utiliser dans mon équipe pour tous les **services d'AA chez Microsoft**. Certains d'entre eux sont intégrés dans le produit de base **Windows** et dans certains produits **Office**._"
```
Not sure if the abbreviation **AA** (Apprentissage Automatique) is really understandable in French but it seems that [Wikipedia](https://fr.wikipedia.org/wiki/Apprentissage_automatique), for example, uses it.

Alternative turn of phrase:  pour tous les services d'AA de mon équipe chez **Microsoft** 🤔

Reply:
I understand your point of view, but I think that **ML** is more known than **AA** for French audience (even if it is not translated from English to French).
As a consequence, If I read **AA** in this sentence, I have to look for a definition to finally understand that it deals with **ML**.

Finally, I will translate it like this:
"_[...] J'utilise énormément **FastAPI** ces jours-ci. [...] En fait, je compte l'utiliser dans mon équipe pour tous les **services de ML chez Microsoft**. Certains d'entre eux sont intégrés dans le produit de base **Windows** et dans certains produits **Office**._"

Thanks for your translation which is better...

Reply:
Totally agree with you @flyusfly 
Let's go for: 
```suggestion
"_[...] J'utilise énormément **FastAPI** ces jours-ci. [...] En fait, je compte l'utiliser dans mon équipe pour tous les **services de ML chez Microsoft**. Certains d'entre eux sont intégrés dans le produit de base **Windows** et dans certains produits **Office**._"
```